### DISCUSSION TITLE
OperaGX version

### DISCUSSION THREAD

**Discussion by MicroSRB2** [dimdenGD/OldTwitter#411] [Ideas]
I don't use Chrome or Edge because i don't want Google spying on me. I use GX because it just runs faster. So i was wondering if there was any way to make a OGX version of the plugin for us gamers. I hate the rate limit as much as the next guy.

**Comment by dimdenGD** [comment]
Opera is Chromium browser, you can install extension from https://chrome.google.com/webstore/detail/old-twitter-layout-2023/jgejdcdoeeabklepnkdbglgccjpdgpmf and it'll work
(Also you still get spied by Google and Opera corp on top with Opera)
